pyanaconda/anaconda_logging.py

```diff
@@ -29,6 +29,10 @@
 from pyanaconda.core import constants
 from pyanaconda.core.path import set_mode
 
+import gi
+gi.require_version("GLib", "2.0")
+from gi.repository import GLib
```

Comment:
Please use `from core.glib import ...` with appropriate change in the `core.glib`.